src/interface.jl

```diff
@@ -1,4 +1,4 @@
-import Base: inv, ∘
+import Base: inv, ∘, ==
```

Comment:
IMO one shouldn't import `==` (or `inv`...) when overloading these methods in multiple source files. In general, I think it is much easier to see which methods are internal and which overload definitions in other packages (and also in which package) if one just defines them explicitly, such as `Base.:(==)(x, y)` in this case.

Reply:
Not opposed to that :+1: